caas/kubernetes/provider/bootstrap.go

```diff
@@ -230,7 +235,9 @@ func (c controllerStack) doCleanUp() {
 // Deploy creates all resources for controller stack.
 func (c controllerStack) Deploy() (err error) {
 	// creating namespace for controller stack, this namespace will be removed by broker.DestroyController if bootstrap failed.
-	if err = c.broker.createNamespace(c.broker.GetCurrentNamespace()); err != nil {
+	nsName := c.broker.GetCurrentNamespace()
```

Comment:
make the namespace = "controller-<controllerName>"

Reply:
changed controller namespace to `controller-<controller-name>`.